jme3-core/src/plugins/java/com/jme3/material/plugins/J3MLoader.java

```diff
@@ -659,7 +659,11 @@ private void readTechnique(Statement techStat) throws IOException{
 
         if(isUseNodes){
             //used for caching later, the shader here is not a file.
-            
+
+            if (technique.getWorldBindings() == null) {
+                technique.setWorldBinds(new ArrayList<UniformBinding>());
```

Comment:
Not sure why we need this? isn't the collection initialized when there is a world bind?
Seems the setter in techniqueDef has only been implemented for this use... are you using it somewhere else?

Reply:
@Nehon I have NPE with access to this collection when I don't have world bindings on technique definition(for example when I want to save shader node material during editing it, when I removed all world bindings(temporary)). For example, you can look at line 58 in the class J3mdTechniqueDefWriter, we don't check null here. I think we don't work with a lot of instances of technique definitions, this optimization isn't necessary to have null here by default.

Reply:
well in that case why not just instantiate the list when it's declared instead of adding a setter?

Reply:
@Nehon I agree, I will update it ;)